mdbenchmark/mdengines/utils.py

```diff
@@ -101,20 +101,29 @@ def analyze_benchmark(engine, benchmark):
     threads = np.nan
     hyperthreading = np.nan
     module = None
+    multidir = np.nan
 
     # search all output files
     output_files = glob(
         os.path.join(benchmark.relpath, PARSE_ENGINE[engine.NAME]["analyze"])
     )
     if output_files:
```

Comment:
I realize this line is not part of the PR, but we can actually remove the `if`-clause. The `for`-loop on the next line will not be executed on an empty list (which is returned when `glob` does not find any matches).